scripts/common.py

```diff
@@ -144,6 +148,21 @@ def get_current_branch_name():
     return git_status_first_line[len(branch_message_prefix):]
 
 
+def get_current_release_version(release_branch_name):
+    """Gets the release version given a release branch name.
+
+    Args:
+        release_branch_name: str. The name of release branch.
+
+    Returns:
+        str. The version of release.
+    """
+    release_version = release_branch_name[len(RELEASE_BRANCH_NAME_PREFIX):]
```

Comment:
Optional: If possible it would be cleaner to do things more semantically rather than using manual operations with indexes. For example I wonder if you can compare the branch name to a regex instead, and use that to extract the relevant "pieces" from it.

(This is optional, don't spend too much time on it. But it would improve readability.)

Reply:
Done.